applications_tests/lethe-fluid/refinement_at_frequency_on.prm

```diff
@@ -0,0 +1,83 @@
+set dimension = 2
+
+subsection simulation control
+  set method           = bdf1
+  set time step        = 0.1
+  set time end         = 0.5
+  set output frequency = 0
+  set output name      = output
+end
+
+subsection physical properties
+  set number of fluids = 1
+  subsection fluid 0
+    set kinematic viscosity = 1
+  end
+end
+
+subsection mesh
+  set type               = dealii
+  set grid type          = hyper_cube
+  set grid arguments     = -1 : 1 : true
+  set initial refinement = 6
+end
+
+subsection boundary conditions
+  set number = 4
+  subsection bc 0
+    set id   = 0
+    set type = noslip
+  end
+  subsection bc 1
+    set id   = 1
+    set type = noslip
+  end
+  subsection bc 2
+    set id   = 2
+    set type = noslip
```

Comment:
Not related to this PR specifically (can coming from someone who never uses the CFD solvers) but "noslip" shouldn't be written "no_slip" ?

Reply:
Yeah or even "no slip". I'm not sure if there is a reason why it is like this, but it could be changed

Reply:
could be no_slip indeed. It's an historic thing